christfellowshipchurch/web-app-v2
# Calls to Action for Content Collections

### About
This PR enables content managers to add an additional CTA underneath a Content Collection from Rock. You'll see that each Feature has been added `callToAction` attribute.

### Test Instructions
* run web locally and test with the [api branch](https://github.com/christfellowshipchurch/services/pull/59) if it hasn't been merged yet
* go to `/testing-content` to see Call to Action underneath a feature.

### Screenshots
![image](https://github.com/christfellowshipchurch/web-app-v2/assets/46049974/f03f58fb-f5fc-401b-aa3c-ee4d1ecfb4b3)

### Closes Tickets
[CFDP-2660](https://christfellowshipchurch.atlassian.net/browse/CFDP-2660)
